tbdex/offering/offering.go

```diff
@@ -55,15 +59,27 @@ type OfferingPaymentMethod struct {
 	Max                    string `json:"max,omitempty"`
 }
 
-// OfferingPayinMethod is an alias for PaymentMethod.
-type OfferingPayinMethod = OfferingPaymentMethod
-
-// OfferingPayoutMethod contains all the fields from PaymentMethod, in addition to estimated settlement time.
-type OfferingPayoutMethod struct {
-	OfferingPaymentMethod
+// PayoutMethod contains all the fields from PaymentMethod, in addition to estimated settlement time.
+type PayoutMethod struct {
+	Kind                    string `json:"kind"`
+	Name                    string `json:"name,omitempty"`
+	Description             string `json:"description,omitempty"`
+	Group                   string `json:"group,omitempty"`
+	RequiredPaymentDetails  string `json:"requiredPaymentDetails,omitempty"` // TODO: change to JSON Schema type
```

Comment:
> // TODO: change to JSON Schema type

Are we going to add a json schema type to web5-go or is there an existing go package we'll use directly here?